python/feathub/processors/local/ast_evaluator/local_ast_evaluator.py

```diff
@@ -70,7 +70,7 @@ def eval_compare_op(self, ast: CompareOp, variables: Optional[Dict]) -> Any:
             return left_value > right_value
         elif ast.op_type == ">=":
             return left_value >= right_value
-        elif ast.op_type == "==":
+        elif ast.op_type == "==" or ast.op_type == "=":
```

Comment:
Can you explain why we need to support both `==` and `=` for the same logic expression? Is there any popular DSL language `e.g. SQL` that does this?

Reply:
I think SQL, both [FlinkSQL](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/functions/systemfunctions/#comparison-functions) or [MySQL](https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html) use and only use `=` for equal comparison. `=` is added so that it is compatible with SQL.

For `==`, I think we can keep it as it is not used in SQL. What do you think?

Reply:
Do you mean we should keep `==`?

Since we want to be compatible with Flink SQL, would it be simpler to remove `=` from the FeatHub expression language so that we have a consistent way to express the equal logic?

Reply:
I agree it is cleaner to only support `=` and remove `==` in Feathub DSL.